apps/condo/domains/banking/schema/BankAccount.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * Generated by `createschema banking.BankAccount 'organization:Relationship:Organization:CASCADE; tin:Text; country:Text; routingNumber:Text; number:Text; currency:Text; approvedAt?:DateTimeUtc; approvedBy?:Text; importId?:Text; territoryCode?:Text; bankName?:Text; meta?:Json; tinMeta?:Json; routingNumberMeta?:Json'`
+ */
+
+const { get } = require('lodash')
+
+const { Text, DateTimeUtc, Select } = require('@keystonejs/fields')
+const { Json } = require('@condo/keystone/fields')
+const { GQLListSchema } = require('@condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@condo/keystone/plugins')
+const { dvAndSender } = require('@condo/domains/common/schema/plugins/dvAndSender')
+
+const access = require('@condo/domains/banking/access/BankAccount')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+const { IMPORT_ID_FIELD, CURRENCY_CODE_FIELD } = require('@condo/domains/common/schema/fields')
+const { COUNTRIES } = require('@condo/domains/common/constants/countries')
+
+const { validateTin } = require('@condo/domains/banking/utils/validate/tin.utils')
+const { validateRoutingNumber } = require('@condo/domains/banking/utils/validate/routingNumber.utils')
+const { validateNumber } = require('@condo/domains/banking/utils/validate/number.utils')
+
+const BankAccount = new GQLListSchema('BankAccount', {
+    schemaDoc: 'Bank account. We receive all transactions from this account and let the user mark them with a certain set of categories!',
+    fields: {
+        organization: ORGANIZATION_OWNED_FIELD,
+
+        tin: {
+            schemaDoc: 'Tax Identification Number',
+            type: Text,
+            isRequired: true,
+            hooks: {
+                validateInput: ({ existingItem, resolvedData, addFieldValidationError }) => {
+                    const newItem = { ...existingItem, ...resolvedData }
+
+                    const country = get(newItem, 'country')
+                    const tin = get(newItem, 'tin')
+
+                    const { result, errors } = validateTin(tin, country)
+
+                    if ( !result ) {
+                        addFieldValidationError(errors[0])
+                    }
+                },
+            },
+        },
+
+        country: {
```

Comment:
To be consistent with fields, related to bank

```suggestion
       bankCountry: {
```

Reply:
I think it's redundant. Is this a BankAccount schema?)